projects/adin1300_dual/zed/system_constr.xdc

```diff
@@ -0,0 +1,69 @@
+set_property -dict {PACKAGE_PIN G16  IOSTANDARD LVCMOS25} [get_ports {rgmii_rxd_b[1]}]            ; ## H23 FMC_LPC_LA19_N
+set_property -dict {PACKAGE_PIN G20  IOSTANDARD LVCMOS25} [get_ports {rgmii_rxd_b[2]}]            ; ## G21 FMC_LPC_LA20_P
+set_property -dict {PACKAGE_PIN G21  IOSTANDARD LVCMOS25} [get_ports {rgmii_rxd_b[3]}]            ; ## G22 FMC_LPC_LA20_N
+set_property -dict {PACKAGE_PIN C22  IOSTANDARD LVCMOS25 SLEW FAST} [get_ports rgmii_txc_b]       ; ## G28 FMC_LPC_LA25_N
+set_property -dict {PACKAGE_PIN A18  IOSTANDARD LVCMOS25 SLEW FAST} [get_ports rgmii_tx_ctl_b]    ; ## H28 FMC_LPC_LA24_P
+set_property -dict {PACKAGE_PIN E19  IOSTANDARD LVCMOS25 SLEW FAST} [get_ports {rgmii_txd_b[0]}]  ; ## H25 FMC_LPC_LA21_P
+set_property -dict {PACKAGE_PIN E20  IOSTANDARD LVCMOS25 SLEW FAST} [get_ports {rgmii_txd_b[1]}]  ; ## H26 FMC_LPC_LA21_N
+set_property -dict {PACKAGE_PIN G19  IOSTANDARD LVCMOS25 SLEW FAST} [get_ports {rgmii_txd_b[2]}]  ; ## G24 FMC_LPC_LA22_P
+set_property -dict {PACKAGE_PIN F19  IOSTANDARD LVCMOS25 SLEW FAST} [get_ports {rgmii_txd_b[3]}]  ; ## G25 FMC_LPC_LA22_N
+
+set_property -dict {PACKAGE_PIN A16  IOSTANDARD LVCMOS25 PULLUP true} [get_ports mdio_fmc_b]      ; ## H31 FMC_LPC_LA28_P
+set_property -dict {PACKAGE_PIN A17  IOSTANDARD LVCMOS25} [get_ports mdc_fmc_b]                   ; ## H32 FMC_LPC_LA28_N
+
+#
+# set_property -dict {PACKAGE_PIN B19  IOSTANDARD LVCMOS25} [get_ports gp_clk_b]                  ; ## D20 FMC_LPC_LA17_CC_P
+
+set_property -dict {PACKAGE_PIN J17  IOSTANDARD LVCMOS25} [get_ports reset_b]                     ; ## H20 FMC_LPC_LA15_N
+set_property -dict {PACKAGE_PIN D22  IOSTANDARD LVCMOS25} [get_ports link_st_b]                   ; ## G27 FMC_LPC_LA25_P
+set_property -dict {PACKAGE_PIN D15  IOSTANDARD LVCMOS25} [get_ports int_n_b]                     ; ## D24 FMC_LPC_LA23_N
+set_property -dict {PACKAGE_PIN E15  IOSTANDARD LVCMOS25} [get_ports led_0_b]                     ; ## D23 FMC_LPC_LA23_P
+set_property -dict {PACKAGE_PIN F18  IOSTANDARD LVCMOS25} [get_ports led_bl_c_c2m]                ; ## D26 FMC_LPC_LA26_P
+set_property -dict {PACKAGE_PIN E18  IOSTANDARD LVCMOS25} [get_ports led_bl_a_c2m]                ; ## D27 FMC_LPC_LA26_N
+set_property -dict {PACKAGE_PIN J20  IOSTANDARD LVCMOS25} [get_ports led_br_c_c2m]                ; ## G18 FMC_LPC_LA16_P
+set_property -dict {PACKAGE_PIN K21  IOSTANDARD LVCMOS25} [get_ports led_br_a_c2m]                ; ## G19 FMC_LPC_LA16_N
+
+
+create_clock -name rx_clk_1       -period  8.0 [get_ports rgmii_rxc_a]
+create_clock -name rx_clk_2       -period  8.0 [get_ports rgmii_rxc_b]
+create_clock -name ref_clk_125    -period  8.0 [get_ports ref_clk_125_p]
+
+#set_property CLOCK_DEDICATED_ROUTE FALSE [get_nets i_system_wrapper/system_i/gmii_to_rgmii_1/U0/rgmii_rxc_ibuf_i/O]
+set_property UNAVAILABLE_DURING_CALIBRATION TRUE [get_ports rgmii_rxc_b]
```

Comment:
This only applies to Ultrascale

Reply:
The ultrascale specific constraint was removed.